src/openrct2/rct2/S6Importer.cpp

```diff
@@ -1382,6 +1385,17 @@ class S6Importer final : public IParkImporter
         }
 
         dst->TrackLocation = { src->track_x, src->track_y, src->track_z };
+        if (dst->track_type && (src->track_type >> 2) == TrackElemType::RotationControlToggleAlias)
+        {
+            // Merging hacks mean the track type that's appropriate for the ride type is not necessarily the track type the ride
+            // is on. It's possible to create unwanted behavior if a user layers spinning control track on top of booster track
+            // but this is unlikely since only two rides have spinning control track - by default they load as booster
+            TileElement* tileElement2 = map_get_track_element_at_of_type_seq(
+                dst->TrackLocation, TrackElemType::RotationControlToggle, 0);
+
+            if (tileElement2 != nullptr)
+                dst->track_type = (TrackElemType::RotationControlToggle << 2) | (src->track_direction & 3);
```

Comment:
I'd like you to add getters and setters for track type and direction to both our tile element and to the RCT12 ones, so that we don't have to hardcode this.

Reply:
This is vehicle import code?

Reply:
Yeah, I didn't say it correctly. But the request still stands (cross out tile element and substitute `Ride` and `RCT2Ride`).